src/react/projects/spark-react/src/SprkDropdown/SprkDropdown.js

```diff
@@ -95,10 +97,10 @@ class SprkDropdown extends Component {
     const { choiceItems, isOpen, triggerText } = this.state;
     return (
       <div ref={this.dropdownRef}>
-        <a
-          className={classNames(
-            'sprk-b-Link',
-            'sprk-b-Link--plain',
+        <SprkLink
+          element="a"
+          variant="plain"
+          additionalClasses={classNames(
```

Comment:
If we want this to be a link with href="#" and our sprklink now defaults href to # then we can remove the href="#nogo" on here.